config/install/core.entity_view_display.node.stanford_event.default.yml

```diff
@@ -185,7 +186,7 @@ third_party_settings:
                 label: hidden
                 type: smartdate_custom
                 settings:
-                  date_format: 'l, F j, Y'
+                  date_format: 'l, F j, Y '
```

Comment:
Is this for the space between spanning dates? if so, this will need to be captured in each profile

Reply:
Yes it is. Ok.